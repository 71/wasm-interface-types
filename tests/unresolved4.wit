;; parse-fail

(module
  (@interface func
    call-core $foo))
