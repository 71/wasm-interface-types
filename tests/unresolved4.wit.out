failed to find func named `$foo`
     --> ../../tests/unresolved4.wit:5:15
      |
    5 |     call-core $foo))
      |               ^
